test/TestConfig/scripts/testKitGen/testKitGen.pl

```diff
@@ -65,21 +66,26 @@
 	elsif ( $argv =~ /^\-\-ottawaCsv=/) {
 		($ottawacsv) = $argv =~ /^\-\-ottawaCsv=(.*)/;
 	}
+	elsif ( $argv =~ /^\-\-buildList=/) {
+		($buildList) = $argv =~ /^\-\-buildList=(.*)/;
+	}
 	else {
 		print
```

Comment:
Might be good to have a line or two:
print "Usage:\n" . perl testKitGen.pl . "--graphSpecs=[example|where to find all possible values]  --javaVersion=[SE80|SE90|...] --impl=[openj9|hotspot|sap] [--output=default] [--projectRootDir=default] [--modesXml=default] [--csv=default] [--buildlist=default]"

Reply:
Thanks, updated. I figured that maybe "perl testKitGen.pl --graphSpecs=[linux_x86-64|linux_x86-64_cmprssptrs|...] --javaVersion=[SE80|SE90|...] --impl=[openj9|hotspot|sap] [options]" is enough, since all other optional options will be shown in the options section below, let me know if this works.